modules/imap/output_modules.php

```diff
@@ -787,6 +788,7 @@ protected function output() {
             $res .= '<input type="hidden" class="move_to_string1" value="'.$this->trans('Move to ...').'" />';
             $res .= '<input type="hidden" class="move_to_string2" value="'.$this->trans('Copy to ...').'" />';
             $res .= '<input type="hidden" class="move_to_string3" value="'.$this->trans('Removed non-IMAP messages from selection. They cannot be moved or copied').'" /></div>';
+            $res .= '<input type="hidden" class="auto_advance_email" name="auto_advance_email" value="1" />';
```

Comment:
Should i create it as a variable in the global scope like the screenshot below or should i create it as a function ?
Because everything in that file is structured in a function.
![CleanShot 2024-02-15 at 17 44 55@2x](https://github.com/cypht-org/cypht/assets/14950700/21cd40f8-49e0-4de6-abae-3744dea45a17)